src/lib.rs

```diff
@@ -17,11 +17,8 @@ mod varint;
 pub(crate) mod max_size;
 
 /// The schema types and macros.
-#[cfg(not(feature = "experimental-derive"))]
-mod schema;
-
 #[cfg(feature = "experimental-derive")]
-pub mod schema;
+pub(crate) mod schema;
```

Comment:
This is partially reverting #134 because I don't think the intent was to expose `schema` publicly until it goes out of experimental. The warning here was about documentation, which should be duplicated if we want to control the visibility from the `cfg`.

Reply:
I will likely intend to take this out of experimental with #128 coming up soon, so it works for me.